File: app/src/main/java/projects/csce/evence/di/loggedin/LoggedInSubComponent.java
package projects.csce.evence.di.loggedin;

import dagger.Subcomponent;
import projects.csce.evence.view.SecondActivity;

@SignedInScope
@Subcomponent(modules = {LoggedInModule.class})
public interface LoggedInSubComponent
{
    void inject(SecondActivity activity);
}

File: app/src/main/java/projects/csce/evence/di/loggedin/LoggedInSubComponent.kt
package projects.csce.evence.di.loggedin

import dagger.Subcomponent
import projects.csce.evence.view.ui.SecondActivity

@SignedInScope
@Subcomponent(modules = [LoggedInModule::class])
interface LoggedInSubComponent
{
	fun inject(activity: SecondActivity?)
}